CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Added
+- Added span tag option to react version of Title kit ([#944](https://github.com/powerhome/playbook/pull/944) @coleerikson)
+
+## [v6.0.1] 2020-7-30
+
+### Fixed
+- Updated failing spec test after update ([#940](https://github.com/powerhome/playbook/pull/940) @jasoncypret)
+
```

Comment:
I think that it's only supposed to contain your changes, not anyone else's.

Reply:
Back in action! Thanks : )